Tasks/password.js

```diff
@@ -1,13 +1,14 @@
+'use strict';
 // Generate random password
 
-let GeneratePassword = (alphabet, length) => {
-  const MAX = alphabet.length;
-  key = '';
+const generatePassword = (alphabet, length) => {
+  const max = alphabet.length;
+  let password = '';
   for (let i = 0; i < length; i++) {
-    Index = Math.floor(Math.random() * MAX);
-    key = key + alphabet[Index];
+    const index = Math.floor(Math.random() * max);
```

Comment:
It's better to extract `random` as a separate function

Reply:
was fixed here https://github.com/kovaliovev/Exams/blob/exams/Tasks/password.js